tests/Tests.AzureAppConfiguration/RefreshTests.cs

```diff
@@ -854,6 +876,219 @@ Response<ConfigurationSetting> GetIfChanged(ConfigurationSetting setting, bool o
             Assert.Equal("newValue", config["TestKey3"]);
         }
 
+        [Fact]
+        public void RefreshTests_RefreshAllTrueForOverwrittenSentinelUpdatesEntireConfiguration()
+        {
+            var serviceCollection = new List<ConfigurationSetting>(_kvCollection);
```

Comment:
We may want to use a different name for this variable. `ServiceCollection` is a well-known class in .NET.
[https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.dependencyinjection.servicecollection?view=dotnet-plat-ext-5.0](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.dependencyinjection.servicecollection?view=dotnet-plat-ext-5.0)

Reply:
I am using `serviceCollection` as the variable name here in order to maintain consistency with all other tests in this class.

Reply:
I don't see any problem fixing it in just this new test.